app/config.py

```diff
@@ -114,6 +122,7 @@ def __init__(self, runtime_environment):
             "max_poll_interval_ms": int(os.environ.get("KAFKA_CONSUMER_MAX_POLL_INTERVAL_MS", "300000")),
             "session_timeout_ms": int(os.environ.get("KAFKA_CONSUMER_SESSION_TIMEOUT_MS", "10000")),
             "heartbeat_interval_ms": int(os.environ.get("KAFKA_CONSUMER_HEARTBEAT_INTERVAL_MS", "3000")),
+            **self.kafka_consumer_ssl,
```

Comment:
... And the values would be set here, instead of doing a var expansion:

```suggestion
            "security_protocol": os.environ.get("KAFKA_SECURITY_PROTOCOL", "PLAINTEXT"),
            "ssl_cafile": os.environ.get("KAFKA_SSL_CAFILE"),
```